### ISSUE TITLE
Table headers are not readable with "Dark" option

### ISSUE THREAD

**Issue by jbickar** [SU-SWS/stanford_light#67]
1. Set Stanford Light as the default theme
1. Select the "Dark" option under "Choose styles" at `admin/appearance/settings/stanford_light`
1. Create a table in HTML and include `th` elements
1. See that the `background-color` is `#f5f5f5` and the `color` is `#fbfbf9`

![2016-04-05-9429-screenshot](https://cloud.githubusercontent.com/assets/821106/14290194/ec5a1b26-fb12-11e5-8a63-8b26ada4f97e.jpg)
